app/services/articles/feed.rb

```diff
@@ -43,5 +43,23 @@ def default_home_feed(user_signed_in: false)
       _featured_story, stories = default_home_feed_and_featured_story(user_signed_in: user_signed_in)
       stories
     end
+
+    def optimized_feed(user, test_variant)
```

Comment:
We should just call this `ABTestFeed` for clarity!

Reply:
Makes sense